.circleci/config.yml

```diff
@@ -643,10 +643,28 @@ jobs:
                 -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=On
           working_directory: /home/circleci/project/build
       - run:
-          name: Build pika Arm64
+          name: Building pika Arm64
           command: |
             ninja -k 0 pika
           working_directory: /home/circleci/project/build
+      - run:
+          name: Building tests Arm64
+          command: |
+            ninja -k 0 tests
+          working_directory: /home/circleci/project/build
+      - run:
+          name: Running tests Arm64
```

Comment:
Don't forget to add the ctest test support (it's the various xml stuff that is in the main workflow). This lets CircleCI give test statistics.